tests/sentry/coreapi/tests.py

```diff
@@ -284,6 +284,42 @@ def test_extra_as_string(self):
         })
         assert data['extra'] == {}
 
+    def test_release_tag_max_len(self):
+        release_key = u'sentry:release'
+        release_value = ('a' * VERSION_LENGTH)
```

Comment:
Added a test to check that the max len of release is accepted. Fine as long as `VERSION_LENGTH <= MAX_CUL
PIRT_LENGTH`